lib/MooX/Options/Descriptive/Usage.pm

```diff
@@ -134,6 +134,10 @@ sub option_text {
     my @message;
     my $lf = _get_line_fold();
     for my $opt(@$getopt_options) {
+        if ($opt->{desc} eq 'spacer') {
```

Comment:
Thats something Getopt::Long::Descriptive does to indicate this $opt is not an option -- its whitespace or text.